website/content/en/preview/concepts/deprovisioning.md

```diff
@@ -129,6 +129,15 @@ data:
   ...
 ```
 
+#### Permissions
```

Comment:
I wonder if this is the right place to document this. Maybe we need a separate page where we document all of the permissions that karpenter needs?

Reply:
Might make sense. Have a low-level detailing of permissions that is stored in a `Reference` section in the docs. Since this was a specific feature that required a specific set of additional permissions, I was thinking it might make sense to include it here in the short-term.

Reply:
What are your thoughts on including this in this section of the docs in the short-term and then moving this over to a separate Reference section in the docs that includes all the details of which features need which permissions?

Reply:
I can add a task to create this reference section.

Reply:
preference for users to discover this in another way, rather than teaching them to discover the permissions here.

Reply:
Not sure if I understand what you mean,, "discover this in another way"? Are you referring to a permissions reference section? If it's not documented, it's not clear to me how the users are going to see a correlation between the permissions they need to set in IAM and the features that are backing these permissions.